packages/core/src/Site/TemplateConfig.ts

```diff
@@ -0,0 +1,61 @@
+import { SiteConfigPage } from './SiteConfig';
+
+type NjkVars = {
+  variableName: string,
+  fileSubstitutes: string[]
```

Comment:
Why is an array used here? There should only be one file for each substitution
Changes below shows that the last file will be used anyway.

Reply:
I used an array so that possible variations of file names that could be used as substitutes.
Kind of like how `['README.md', 'Home.md']` is used in `addIndexPage()`
([link](https://github.com/WillCWX/markbind/blob/053d3b85a37744232396eae3b73012eb3ff2e6c0/packages/core/src/Site/index.ts#L396))

Though the current nunjucks variables only have one file name as a substitute, I thought that using an array may allow a greater range of possible file names. (at the cost of some complexity and readability)

Reply:
Ah I see where you are coming from. That;s ok then :) 
Also could you change the variable name to something more obvious that its just for files? Eg. `NjkFileVars`